src/features/schedule/components/CalendarItemComponent.tsx

```
@@ -0,0 +1,386 @@
+  // ドラッグ終了時の処理
+  const handleDragStop: DraggableEventHandler = (e, data) => {
+    setIsDragging(false);
+
+    // 移動量を15分単位に丸める
+    const quarterCount = Math.round(data.y / quarterHeight);
+    // 分単位の変化量を計算（15分単位）
+    const diffMinutes = quarterCount * 15;
+
+    // 新しい開始時間と終了時間を計算
+    const updatedStartDateTime = new Date(calendarItemWithMeta.calendarStartDateTime);
+    updatedStartDateTime.setMinutes(updatedStartDateTime.getMinutes() + diffMinutes);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)